tests/regression/74-threadCreate/01-simple.c

```diff
@@ -0,0 +1,32 @@
+// PARAM: --disable ana.thread.context.create-edges
```

Comment:
All these tests disable the new option, so aren't they just testing the old behavior without the edges?
Seems like there should be tests for the cases where enabling the new option actually makes us more precise than before as well.

Reply:
The option is active by default as that corresponds to the old behavior. What the changes in here are about is holding on to more precision, when reducing the amount of context-sensitivity.